.github/components.yml

```diff
@@ -0,0 +1,67 @@
+CPU:
+  dependent_components:
+    Python_API: [build, test]
+    C_API: [build, test]
+
+Python_API:
+  dependent_components:
+    CPU: [build]
```

Comment:
Just to think about.. alternative scheme:
- CPU defines `affected_components` which assumes full revalidation
- Python defines `required_components` which assume only build.

In this case config would look like:
```
CPU:
  affected_components:
    - Python_API
    - C_API
  required_components: # just an example of component which needs to be build
    - ngraph

Python_API:
  required_components:
  - CPU    
```